python/ml4ir/applications/classification/tests/test_classification_serving.py

```diff
@@ -73,7 +73,8 @@ def test_serve_default_pipeline(self):
             tfrecord_signature(protos=tf.gather(sequence_example_protos, [i]))[
                 self.args.output_name
             ]
-            for i in range(self.args.batch_size)
+            for
```

Comment:
[Minor] @mohazahran This is such a strange formatting change haha.

Reply:
That's weird! I don't know how did that happen